### ISSUE TITLE
[Request] Play something or random lists

### ISSUE THREAD

**Issue by Bequiet19** [elgatito/plugin.video.elementum#855]
Is there a possibility to elementum get this function as Netflix. This works as a shufle function in Netflix. I guess this kind of function can't be integrated, but maybe adding random list could be made that would mix movies or tv shows. For example to mix movies or tv shows from trakt or tmdb trending, popular or some other list or to mix movies from some genre... Many trakt or tmdb lists are fixed and when I enter them they are always the same movies on begining. With a random list, it would be great if I went to, for example, to trakt popular list, it would throw out a different movies every time that I hadn't watched. Netflix does the similar thing. I think this would be interesting for many users. Of course, you should not throw out the basic trakt or tmdb lists, but insert a special group with random lists. I apologize for the slightly longer post. I just hope I was clear enough.

**Comment by antonsoroko** [comment]
looks very similar to #802

**Comment by matke-84** [comment]
@antonsoroko Yes, I made it a similar request a few months ago. I still think this list is interesting and I still hope that a miracle may happen. :-)
